cmd/lvmd/app/config.go

```diff
@@ -18,10 +18,13 @@ type Config struct {
 	DeviceClasses []*lvmdTypes.DeviceClass `json:"device-classes"`
 	// LvcreateOptionClasses are classes that define options for the lvcreate command
 	LvcreateOptionClasses []*lvmdTypes.LvcreateOptionClass `json:"lvcreate-option-classes"`
+	// ProfilingPort is the port number to expose profiling. If 0, profiling is disabled.
+	ProfilingPort int `json:"profiling-port"`
 }
 
 var config = &Config{
-	SocketName: topolvm.DefaultLVMdSocket,
+	SocketName:    topolvm.DefaultLVMdSocket,
+	ProfilingPort: 0,
```

Comment:
this is the zero value and you can just leave it out of the definition of the struct